docs/user-quick-start.md

```diff
@@ -149,21 +149,19 @@ enough resources:
    for the details.
    Tested with version v1.12.0.
 
-1. Install the `virtctl` tool. See
-   [virtctl install](https://kubevirt.io/quickstart_minikube/#virtctl)
-   for the details.
-   Tested with version v1.0.1.
-
+1. Install the `virtctl` tool.
+   Working Example:
    ```
-   curl -L -o virtctl https://github.com/kubevirt/kubevirt/releases/download/v1.0.1/virtctl-v1.0.1-linux-amd64
+   curl -L -O https://github.com/kubevirt/kubevirt/releases/download/v1.2.0/virtctl-v1.2.0-linux-amd64
    ```
```

Comment:
Same issue with the linter:

docs/user-quick-start.md:154: MD031 Fenced code blocks should be surrounded by blank lines
docs/user-quick-start.md:156: MD031 Fenced code blocks should be surrounded by blank lines
docs/user-quick-start.md:161: MD031 Fenced code blocks should be surrounded by blank lines